qc_baselib/__init__.py

```diff
@@ -3,7 +3,7 @@
 # Public License, v. 2.0. If a copy of the MPL was not distributed
 # with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-__version__ = "0.1.0"
+__version__ = "v1.0.0-rc.1"
```

Comment:
As far as I know this must be `__version__ = "1.0.0rc1"` (and should be the same string included in poetry) [Reference](https://peps.python.org/pep-0386/#the-new-versioning-algorithm)

Reply:
Reverted to 1.0.0rc1. Tag will be v1.0.0-rc.1.